helper/testhelpers/mssql/mssqlhelper.go

```diff
@@ -13,28 +13,37 @@ import (
 
 const mssqlPassword = "yourStrong(!)Password"
 
+// This constant is used in retrying the mssql container restart, since
+// intermittently the container starts but mssql within the container
+// is unreachable.
+const numRetries = 3
+
 func PrepareMSSQLTestContainer(t *testing.T) (cleanup func(), retURL string) {
 	if os.Getenv("MSSQL_URL") != "" {
 		return func() {}, os.Getenv("MSSQL_URL")
 	}
+	var svc *docker.Service
```

Comment:
May as well keep these loop-scoped since they're not used outside of it.

Reply:
Makes sense, will move those inside the loop!